platform-ui-skin/src/main/webapp/skin/less/helpers.less

```diff
@@ -432,3 +432,6 @@
     }
   }
 }
+.activityFilter {
+  flex: 0 0!important;
+}
```

Comment:
IMO, it shouldn't be added here, it must be added in the dedicated file for the activity stream